infra/makemake/configuration.nix

```
@@ -46,7 +46,11 @@
 
   users = {
     mutableUsers = false;
-    users.root.openssh.authorizedKeys.keys = import ../ssh-keys.nix;
+    users.root.openssh.authorizedKeys.keys = (import ../ssh-keys.nix).root;
+    users.remotebuild = {
+      isNormalUser = true;
+      openssh.authorizedKeys.keys = (import ../ssh-keys.nix).remotebuild;
```

A pattern that is very likely much easier to work with is having the keys in a directory, where each filename is the username. This is just a sketch:
```suggestion
      openssh.authorizedKeys.keyFiles = with lib;
        map (n: ./keys/${n}) (attrNames (builtins.readDir ./remotebuild));
```
You probably want to store those key groups in a `let` binding right here in the file, and combine them as needed, so whatever logic there is visible at first glance.